src/cli/idm/idm-export.js

```diff
@@ -0,0 +1,109 @@
+import { Command, Option } from 'commander';
+import * as common from '../cmd_common.js';
+import { getTokens } from '../../api/AuthApi.js';
+import {
+  exportAllConfigEntities,
+  exportAllRawConfigEntities,
+  exportConfigEntity,
+} from '../../ops/IdmOps.js';
+import storage from '../../storage/SessionStorage.js';
+import { printMessage } from '../../ops/utils/Console.js';
+
+const program = new Command('frodo idm export');
+
+program
+  .description('Export IDM configuration objects.')
+  .helpOption('-h, --help', 'Help')
+  .showHelpAfterError()
+  .addArgument(common.hostArgumentM)
+  .addArgument(common.realmArgument)
+  .addArgument(common.userArgument)
+  .addArgument(common.passwordArgument)
+  .addOption(common.deploymentOption)
+  .addOption(common.insecureOption)
+  .addOption(
+    new Option(
+      '-i, --idm-name <idm-name>',
```

Comment:
I don't want to use the -i flag for this use case. I fell like the existing one or a --type or so would be better suited. We use -i/--<x>-id to identify an object but here we are really identifying an object type or configuration type or name.

But mainly I don't like it because it's a breaking change from the last version.